### ISSUE TITLE
reload/search broken

### ISSUE THREAD

**Issue by shanelecuyer** [theahura/shoot-the-messenger#116]
when the page tries to reload, the NEXT_QUERY fails to find the results div due to the span[role="gridcell"] parameter not returning results. [aria-label="Next"] doesn't exist anymore.

**Comment by shanelecuyer** [comment]
ive submitted pull requests that i believe contain the fix for this

**Comment by SeanBannister** [comment]
Just confirming I'm also experiencing this, the error message returned is:
> Attempting to run search with message: ....
> Could not load search bar. Failing.

**Comment by theahura** [comment]
Left comments on #117 
First, thank you so much for dropping a PR, I really appreciate it. 
Left a quick comment, hopefully shouldn't take too long to improve, but let me know if you think you'll get back to it in a reasonable amount of time (if not, tempted to just merge it :shrug:)

**Comment by theahura** [comment]
Merged your pr
